resources/examples/simple_cluster/inventory/group_vars/all/equipment_all/equipment_profile.yml

```diff
@@ -13,7 +13,7 @@ equipment_profile:
   console:
   kernel_parameters:
 
-  access_control: true # Selinux, AppArmor, etc. Selinux: true = enforcing, false = disabled
+  access_control: enforcing # Selinux, AppArmor, etc. Selinux: enforcing, permissive, disabled
```

Comment:
Can you also provide theses modifications for the multi iceberg inventory example

Reply:
Done